website/versioned_docs/version-5.x/api.md

```diff
@@ -396,20 +396,20 @@ const appsForACertainRoute = singleSpa.checkActivityFunctions({
 console.log(appsForACertainRoute); // ['app2']
 ```
 
-Will call every app's activity function with the `mockWindowLocation` and give you list of which applications should be mounted with that location.
+Will call every app's activity function with `location` and give you list of which applications should be mounted with that location.
```

Comment:
Since it's impossible to construct a `Location` object, I think we should clarify here that you can use a `URL` object here. The examples and documentation should be updated to reflect that, as that's a better pattern than providing the `pathname` object thing shown in the current examples:

```js
const appsForACertainRoute = singleSpa.checkActivityFunctions(new URL(location.href, '/app2'));
```

Reply:
Good point. Done.